db/migrate/20250930015506_remove_unnecessary_indicies.rb

```
@@ -0,0 +1,18 @@
+class RemoveUnneccessaryIndecies < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def up
+    remove_index :team_rating_snapshots,
+                  name: :idx_on_team_id_season_id_snapshot_date_8de7607130,
+                  algorithm: :concurrently
```

RuboCop (1.80.2)

[convention] Align the arguments of a method call if they span more than one line.

(Layout/ArgumentAlignment)